stack/framework/hal/chips/HDC1080DM/HDC1080DM.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __HDC1080DM__H
+#define __HDC1080DM__H
+
+#include "hwi2c.h"
+#include <stdio.h>
+#include "errors.h"
+
+typedef enum {
+	HDC1080_RESOLUTION_8BIT,
+	HDC1080_RESOLUTION_11BIT,
+	HDC1080_RESOLUTION_14BIT,
+} HDC1080_measurement_resolution;
+
+typedef union {
+	uint8_t rawData;
+	struct {
+		uint8_t HumidityMeasurementResolution : 2;
+		uint8_t TemperatureMeasurementResolution : 1;
+		uint8_t BatteryStatus : 1;
+		uint8_t ModeOfAcquisition : 1;
+		uint8_t Heater : 1;
+		uint8_t ReservedAgain : 1;
+		uint8_t SoftwareReset : 1;
+	};
```

Comment:
is this working? I would think it needs to be packed to map correctly

Reply:
Haven't actually used this, so probably wouldn't have worked